hydromt/models/model.py

```diff
@@ -97,10 +98,6 @@ def __init__(
         """
         # Recursively update the options with any defaults that are missing in the configuration.
         components = components or {}
-        components = deep_merge(
-            {"region": {"type": "ModelRegionComponent"}}, components
-        )
-
         self.target_crs = CRS.from_user_input(target_model_crs)
```

Comment:
I'm not sure if we want to have this `target_crs`. Remove and use `region_component.region` instead?

Reply:
I think it's better yes